docs/dev/metrics.md

```diff
@@ -0,0 +1,56 @@
+# CAO Metrics
+
+The Cluster Autoscaler Operator reports the following metrics:
+
+## Metrics provided by the controller runtime
+
+The [controller runtime](https://github.com/kubernetes-sigs/controller-runtime)
+integration with the operator provides several metrics about the internals of the
+operator. You can find more information about these metrics names and their labels through
+the following links:
+
+### Kubernetes controller metrics
+
+These metric names begin with `controller_runtime_reconcile_`. The labels
```

Comment:
Are these metrics actually getting collected?  No CAO/MAO stuff being reported in 4.5.

Reply:
they may not be collected, but they are exposed by the metrics endpoint for both operators.

Reply:
Same here.  Not getting collected by prometheus.  I think some other operators do have this information being collected in prometheus.  In any case, we should specify that we are/aren't collecting these?

Reply:
>  In any case, we should specify that we are/aren't collecting these?

that's a good question, my gut feeling is that we should document all the possible metrics someone could scrape from the operator. given that the collection/telemetry stuff is controlled from a different location, perhaps we should drop a link to that repo so readers can evaluate what is being exported.